src/pyk/kore/rpc.py

```diff
@@ -1133,7 +1133,7 @@ def start(self) -> None:
 
     def close(self) -> None:
         _LOGGER.info(f'Stopping KoreServer: {self.host}:{self.port}, pid={self.pid}')
-        self._proc.send_signal(SIGINT)
+        self._proc.send_signal(SIGTERM)
```

Comment:
```suggestion
        self._proc.terminate()
```

(https://docs.python.org/3/library/subprocess.html#subprocess.Popen.terminate)